homeassistant/components/alexa/capabilities.py

```diff
@@ -695,6 +695,7 @@ class AlexaSpeaker(AlexaCapability):
         "en-US",
         "es-ES",
         "es-MX",
+        "fr-FR",
```

Comment:
I don't see it listed here https://developer.amazon.com/en-US/docs/alexa/device-apis/list-of-interfaces.html